packages/@aws-cdk-testing/framework-integ/test/aws-synthetics/test/integ.canary-sys-nodejs-puppeteer-8.ts

```diff
@@ -0,0 +1,22 @@
+import * as cdk from 'aws-cdk-lib/core';
+import { Canary, Cleanup, Code, Runtime, Schedule, Test } from 'aws-cdk-lib/aws-synthetics';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+const app = new cdk.App();
```

Comment:
nit: an empty line between imports and logic will be better?

Reply:
Thanks. I've added.